components/bl-tabs-component/src/index.js

```diff
@@ -0,0 +1,72 @@
+import { useState, useEffect, useMemo } from 'react';
+
+import { TabControl } from './tab-control';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function TabsComponent({ component, eventHandlers, appData, pageData, parentDataModel, pods }) {
+  const { classList, style, display, disabled, variant, tabs } = component;
+  const { onChange } = eventHandlers;
+
+  const [currentTabId, setCurrentTabId] = useState(null);
+
+  component.getCurrentTabId = () => currentTabId;
+  component.setCurrentTabId = id => {
+    setCurrentTabId(id);
+    onChange({ currentTabId: id });
+  }
+
+  const podsContent = pods["Tabs Content"];
+
+  const tabsList = useMemo(() => {
+    if (Array.isArray(tabs)) {
+      return tabs.filter((tab, index) => {
+        if (tab.id && tab.label) {
+          return true;
+        }
+
+        console.log(`Invalid tab item ${++index}`);
```

Comment:
console.error

`Invalid tab item: {label:..., value:...}` or something readable, but not shifted index.